docs/content/preview/drivers-orms/smart-drivers.md

```diff
@@ -28,6 +25,18 @@ Yugabyte has developed the following smart drivers, available as open source sof
 
 All YugabyteDB smart driver libraries are actively maintained, and receive bug fixes, performance enhancements, and security patches.
 
+## Key features
+
+YugabyteDB smart drivers have the following key features.
+
+| Feature | Notes |
+| :--- | :--- |
+| Multiple hosts | As with the upstream driver (with the exception of node.js), you can specify multiple hosts for the initial connection, thereby avoiding dropped connections in the case where the primary host is unavailable. |
+| [Cluster aware](#cluster-aware-connection-load-balancing) | Smart drivers perform automatic uniform connection load balancing<br/>After the driver establishes an initial connection, it fetches the list of available servers from the cluster and distributes connections evenly across them. |
+| [Servers refresh interval](#servers-refresh-interval) | By default, the driver refreshes the list of available servers every five minutes. The interval is configurable (with the exception of Python). |
+| [Topology aware](#topology-aware-connection-load-balancing) | In cases where you want to restrict connections to specific geographies for lower latency, you can target specific regions and zones, along with fallback zones, across which to balance connections. |
```

Comment:
I would move 'Topology aware' one place above, right after 'Cluster aware'.
'Servers refresh interval' is not really a feature but a way to configure a property used by 'Cluster aware' and 'Topology aware' features.